polkadot/node/core/pvf/src/artifacts.rs

```diff
@@ -16,10 +16,10 @@
 
 //! PVF artifacts (final compiled code blobs).
 //!
-//!	# Lifecycle of an artifact
+//! # Lifecycle of an artifact
```

Comment:
Random thought, could we add an `# Artifact integrity` section here? It could explain that artifacts must remain valid to satisfy the `SAFETY` constraints of `execute_artifact`. Therefore we do two things: check file integrity and version compatibility on host startup, and also before voting against a candidate we re-check the file integrity.

Reply:
How about we do this when we have decided what to do if an artifact is found corrupted? I'll make a note of this.